packages/wallet-core/src/store/types.ts

```diff
@@ -256,6 +258,8 @@ export interface RootState {
   experiments: Partial<Record<ExperimentType, boolean>>;
   whatsNewModalVersion: string;
   enabledChains: WalletIdNetworkMap<ChainId[]>;
+
+  errorLog: LiqualityErrorJSON[];
```

Comment:
Why here? This is the account object in the state. I don't see any errors being specifically stored for the account - only see it at root

Reply:
@monokh Not sure I understand your point. Currently there is one errorLog and it's in the state's root. Do you mean each account should have its own errorLog?